src/offset_date_time.rs

```diff
@@ -40,15 +40,12 @@ pub struct OffsetDateTime {
 }
 
 impl OffsetDateTime {
-    /// Create a new `OffsetDateTime` with the current date and time.
-    ///
-    /// This currently returns an offset of UTC, though this behavior will
-    /// change once a way to obtain the local offset is implemented.
```

Comment:
Would you consider having `now()` continue to return the time in UTC and instead implement a `now_local()`?

Reply:
My thinking behind having `now()` return local by default is that if someone does `OffsetDateTime::now().hour()`, they're more than likely expecting _their_ hour, not UTC.

Equality is implemented in terms of the actual time (UTC), so the two methods will actually return equal values (ignoring the difference in time between calling them).

Reply:
All existing usage of `now()` will be silently broken with this change. In addition, the source of most bugs (in my experience) of code involving timestamps is often related to trying to juggle timezones or converting back and forth. I always recommend storing and using timestamps in UTC and only converting to local time in the final step before presenting to a user. The original implementation (where `now()` was UTC) encouraged this behavior. I fear that making `now()` return local time is a step backwards.

Reply:
The internal usage _is_ stored as UTC. It's only converted to the stored offset as necessary.

How will existing uses of `now()` be broken, given that all equality, comparisons, hashing, etc. are implemented in terms of the timestamp? The _only_ thing changing is the offset.

Reply:
Broken in the sense that prior to the change the offset was UTC and now it's local timezone. I just mean that it's a breaking change. 

> The internal usage is stored as UTC

I understand that. I'm saying that `now() -> UTC` encourages others to also use UTC.

Reply:
It was and is clearly documented that the behavior would change, so anyone relying on that shouldn't be. If they need UTC, they should already be doing `.to_offset(offset!(UTC))`.

I will consider this, however.

Given that I still need eyes to look everything over for safety and someone to ensure everything works on Windows (and have been asking for around a week), I likely have a bit before a decision must be made.

Reply:
Updated to have `now()` return UTC and `now_local()` return local. This is in accordance with the "vision" page in the wiki.

Thanks!